clients/drcachesim/tracer/instru.cpp

```diff
@@ -181,17 +181,70 @@ instru_t::instr_to_prefetch_type(instr_t *instr)
     }
 }
 
+#ifdef AARCH64
+bool
+instru_t::is_aarch64_icache_flush_op(instr_t *instr)
+{
+    if (instr_get_opcode(instr) != OP_sys)
+        return false;
+    switch (opnd_get_immed_int(instr_get_src(instr, 0))) {
+    // i#4328: Handle privileged icache operations.
+    case DR_IC_IVAU: return true;
+    }
```

Comment:
Add a `TODO i#4406: xxxxxx` comment for the missing handling code here (else the reader thinks this is a bug b/c it looks like you think you handled them all).

Reply:
Done.